agent/tests/conftest.py

```diff
@@ -5,7 +5,9 @@
 from click.testing import CliRunner
 from agent import di
 from agent.api import main
+from agent.modules import constants
 
+constants.TEST_ENV = True
```

Comment:
if it's a constant you shouldn't change it in your code
This still should be an environmental variable

p.s. this test/prod env variable is a bad practice in general, maybe there is a way to get rid of it completely, I'll add to the backlog